File: Server_Spring/Elderly_Care/src/main/java/kr/spring/care/board/model/BoardDTO.java
package kr.spring.care.board.model;

import java.util.Date;
import kr.spring.care.user.entity.User;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class BoardDTO {

	private Long num;
	private String title;
	private String writer;
	private String content;
	private Date regdate;
	private Long hitcount;
	private Long replycnt;
	
	private String regDateStr;
	
	//댓글
//	private List<Comment> comments;
	
	private User user;
	private Long userId;
	
	
	

}

File: Client_Android/app/src/main/java/com/example/elderlycare/board/vo/BoardDTO.kt
package com.example.elderlycare.board.vo

import User
import java.util.Date

data class BoardDTO(
    val num: Long = 0,
    var title: String = "",
    var writer: String = "",
    var content: String = "",
    var regdate: Date = Date(),
    var hitcount: Long = 0,
    var replycnt: Long = 0,
    var regdateStr: String = "",
    var userId: Long = 0
)